docker-compose.yml

```diff
@@ -13,23 +13,8 @@ services:
       - 5432
     restart: always
 
-  index:
-     image: opendatacube/datacube-index:latest
-     environment:
-        - DB_HOSTNAME=postgres
-        - DB_USERNAME=postgres
-        - DB_PASSWORD=opendatacubepassword
-        - DB_DATABASE=postgres
-        - DB_PORT=5432
-        - AWS_DEFAULT_REGION=ap-southeast-2
-     volumes:
-        - ./Tests/setup_test_datacube.sh:/usr/local/bin/setup_test_datacube.sh
-     depends_on:
-        - postgres
-     entrypoint: bash -c 'sleep infinity'
-
   sandbox:
-     image: geoscienceaustralia/sandbox:latest
+     image: 538673716275.dkr.ecr.ap-southeast-2.amazonaws.com/geoscienceaustralia/sandbox:latest
```

Comment:
I think this needs to point to the specific Sandbox image that is used on the "Default environment" on Prod Sandbox. E.g. `2.0.0`?

(We're going to have to update this every time we change the Prod Sandbox image, which isn't ideal - I'd love us to have a "stable" tag that we could use to set both the prod Sandbox image and this line too!)

Reply:
The whole point of setting up automated test workflow is to guarantee sandbox image not breaking notebooks in the update, or for us to be notified asap if it does require changes not back compatible. Any release will be based the current `latest` for docker image.  Hence if the test is performed on `latest`, it should be working on any image in the life cycle of `latest -> release -> latest`.

Reply:
That's only one of the two use cases for our tests - they fullfill two different but important purposes on DEA Notebooks vs DEA Sandbox:

1) On DEA Sandbox: to guarantee changes to the Sandbox image don't break notebooks. For this, we need to test against `latest`.

2) On DEA Notebooks: to guarantee changes to Jupyter Notebooks and Python code don't break other code in the DEA Notebooks repo. For this, we need to test against **the same Sandbox image that is used for the "Default environment" on Prod Sandbox.** 

The #2 is really important for us - and testing against `latest` is useless for that because `latest` isn't available to general public users on the DEA Sandbox. We definitely need to run our tests against something like `2.0.0` for DEA Notebook's tests.

Reply:
Can we merge as it is though, for the time being? I have some changes to make to `dea-sandbox` after merging this pr. I'll add new tag together.  Then I can come back again and change here when the image is ready to pull.

Reply:
Yep sounds good, will mark as approved now